README.md

```diff
@@ -29,3 +29,23 @@
     "rest_interval": "7:00"
 }
 ```
+
+#### GET  '/api/rand_exercises'
+
+```
+#REQUEST BODY
```

Comment:
YES documentation is the best!!

Reply:
I'm never saving documentation till the end again!

Reply:
Yall rock for making this a priority - it's SO helpful for us!